test/java/src/test/java/ghostdriver/DirectFileUploadTest.java

```diff
@@ -45,6 +46,7 @@
 import java.io.*;
 import java.util.List;
 
+@Ignore
```

Comment:
> Not working?

correct.  in phantomjs 2.5 `WebPage#uploadFile` needs a native click event to upload the file.  i'm pretty sure that none of the phantomjs clients even use this api.